docker/models/containers.py

```diff
@@ -175,14 +176,79 @@ def exec_run(self, cmd, stdout=True, stderr=True, stdin=False, tty=False,
         Raises:
             :py:class:`docker.errors.APIError`
                 If the server returns an error.
+
+        Example:
```

Comment:
Not sure this is the best place for such a long example, especially since it applies to `attach()` as well.

Reply:
I'm not sure where to put it though. I can just copy paste if for `attach()`, or add a reference to it.

Reply:
Maybe as a separate docs page. We can figure that out later though, it's not worth blocking the PR.